content/enterprise_influxdb/v1.10/administration/configure/config-data-nodes.md

```diff
@@ -736,17 +736,15 @@ Environment variable: `INFLUXDB_HINTED_HANDOFF_RETRY_RATE_LIMIT`
 
 Default is `"1s"`.
 
-The time period after which the hinted handoff retries a write after the write fails.
+The time period after which the hinted handoff retries a write after the write fails. There is an exponential back-off, which starts at 1 second and increases with each failure. It resets on success.
```

Comment:
"... increases with each failure until it reaches `retry-max-interval`.  Retrials will then occur at `retry-max-interval`.  On any successful retrial, the waiting period will be reset to `retry-interval`

Reply:
added the new verbiage